geomesa-kafka/geomesa-kafka-09-datastore/src/main/scala/org/locationtech/geomesa/kafka09/GeoMessage.scala

```diff
@@ -0,0 +1,189 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.kafka09
+
+import java.nio.ByteBuffer
+import java.nio.charset.StandardCharsets
+
+import com.typesafe.scalalogging.LazyLogging
+import org.joda.time.Instant
+import org.locationtech.geomesa.features.SerializationOption.SerializationOptions
+import org.locationtech.geomesa.features.kryo.KryoFeatureSerializer
+import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
+
+sealed trait GeoMessage {
```

Comment:
One big question about the 0.8/0.9 work is: what can be shared?

I think this could be factored out in a common module.  I know that may be different than we initially discussed.

Any ideas how much changed?